extension/popup/popup.css

```diff
@@ -25,7 +25,6 @@ body,
   width: 350px;
 }
 
-.timer #zap-wrapper,
```

Comment:
It looks like this reverts another recent commit (#1360)

Reply:
ok. Let me take an update from git and then I'll commit again.

Reply:
@ianb  I've taken the update from the original repository to may local and make my PR. Please have a look once and let me know in case still there is any issue.

Thanks